src/activities/rahulworld/rahulworld.svg

```diff
@@ -0,0 +1,530 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
there should be a README file telling where the images/icons are taken from (and be sure we have the rights to use them)